Makefile

```diff
@@ -183,7 +183,7 @@ $(CONFIG_DIR)/cluster-relocation.yaml: $(CONFIG_DIR) $(CLUSTER_RELOCATION_TEMPLA
 		$(CLUSTER_RELOCATION_TEMPLATE) > $@
 
 
-site-config.iso: $(CONFIG_DIR)/cluster-relocation.yaml edge_configs/static_network.cfg
+site-config.iso: $(CONFIG_DIR)/cluster-relocation.yaml edge_configs/static_network.cfg ## Create site-config.iso				make site-config.iso CLUSTER_NAME=new-name BASE_DOMAIN=foo.com
```

Comment:
This comment seems redundant, as the purpose of any makefile target is to create a file named after said target

Targets are recipes on how to create files

Unless their phony targets, which means they're not creating any file

Reply:
They're used to auto-generate a help when running `make help`

Reply:
Ack